### ISSUE TITLE
Undefined name 'cflags' in xcode_emulation.py

### ISSUE THREAD

**Issue by cclauss** [nodejs/node-gyp#1772]
https://travis-ci.com/nodejs/node-gyp/jobs/206213800#L223

https://github.com/nodejs/node-gyp/blob/master/gyp/pylib/gyp/xcode_emulation.py#L847

@refack Your advise please.

* **Node Version**: 
* **Platform**: 
* **Compiler**: 
* **Module**: 

<details><summary>Verbose output (from npm or node-gyp):</summary>

```
Paste your log here, between the backticks. It can be:
  - npm --verbose output,
  - or contents of npm-debug.log,
  - or output of node-gyp rebuild --verbose.
```

</details>

**Comment by bnoordhuis** [comment]
Ping @refack

**Comment by cclauss** [comment]
Closed in #1901
